src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/Conversions/ConversionHelpers.cs

```diff
@@ -37,7 +37,7 @@ public static AzureRmRecoveryServicesContainerBase GetContainerModel(ProtectionC
                 {
                     containerModel = new AzureRmRecoveryServicesIaasVmContainer(protectionContainer);
                 }
-                if (protectionContainer.Properties.GetType().IsSubclassOf(typeof(MabProtectionContainer)))
+                if (protectionContainer.Properties.GetType().Name == typeof(MabProtectionContainer).Name)
```

Comment:
You don't need to use ".Name". Just equate GetType and typeof(..). That's stronger and correct.